Sara/index.js

```diff
@@ -0,0 +1,10 @@
+// 1. Write program to picking random country from countries.
+let randomCountry = ['Germany', 'Italy', 'USA'] ;
+console.log('randomCountry ==>', randomCountry[Math.floor(Math.random() * randomCountry.length)]) ;
+
+// 2. Write program to print random name from names in the following format 'Hallo ' and the random name.
+let randomName = ['Alex', 'Max', 'Tommy'] ;
+console.log('randomName ==>', 'Hello ' + randomName[Math.floor(Math.random() * randomName.length)]) ; // Hello Alex
+
+// 3. Write program to picking number name from numbers. The output Should be absolute (not negative numbers).
```

Comment:
Here is the solution of 3rd task!
let numbers = "-1 7 -33 -8 9";
let sep = numbers.split(" ");
let pickNumber = sep[Math.floor(Math.random() * 5)];
console.log("picking a number ==>", Math.abs(pickNumber));